"Versionless" -> "Release Tracks" (#6471)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/cloud-release-tracks.md

---
title: "Release tracks in dbt Cloud"
sidebar_label: "dbt Cloud Release Tracks"
description: "Learn how to get automatic upgrades to dbt in dbt Cloud. Access new features and enhancements as soon as they become available."
---

Since May 2024, new capabilities in the dbt framework are delivered continuously to dbt Cloud. Your projects and environments are upgraded automatically on a cadence that you choose, depending on your dbt Cloud plan.

Previously, customers would pin to a minor version of dbt Core, and receive only patch updates during that specific version's active support period. Release tracks ensure that your project stays up-to-date with the modern capabilities of dbt Cloud and recent versions of dbt Core.

This will require you to make one final update to your current jobs and environments. When that's done, you'll never have to think about managing, coordinating, or upgrading dbt versions again.

> By moving your environments and jobs to release tracjs you can get all the functionality in dbt Cloud as soon as it's ready. On the "Latest" release track, this includes access to features _before_ they're available in final releases of dbt Core OSS.

Reviewer: ```suggestion
By moving your environments and jobs to release tracks you can get all the functionality in dbt Cloud as soon as it's ready. On the "Latest" release track, this includes access to features _before_ they're available in final releases of dbt Core OSS.
```

## Which release tracks are available?

> - **"Latest"** (available to all plans, formerly called "Versionless"): Get all the latest functionality in dbt Cloud, continuously. Includes early access to new features of the dbt framework before they're available in open source releases of dbt Core.

Reviewer: ```suggestion
- **"Latest"** (available to all plans, formerly called "Versionless"): Provides a continuous release of the latest functionality in dbt Cloud. Includes early access to new features of the dbt framework before they're available in open source releases of dbt Core.
```

> - <Lifecycle status="coming soon"/> **"Compatible"** (available to Team + Enterprise): Monthly release aligned to the most recent open source versions of dbt Core and adapters, plus additional functionality that's available exclusively in dbt Cloud.

Reviewer: I wonder if we should say "compatible with the most recent..."
```suggestion
- <Lifecycle status="coming soon"/> **"Compatible"** (available to Team + Enterprise): Provides a monthly release aligned with the most recent open source versions of dbt Core and adapters, plus functionality exclusively available in dbt Cloud.
```

> - <Lifecycle status="coming soon"/> **"Compatible"** (available to Team + Enterprise): Provides a monthly release aligned with the most recent open source versions of dbt Core and adapters, plus functionality exclusively available in dbt Cloud.

Reviewer 2: love this coming soon lifecycle pill 😍

> - <Lifecycle status="coming soon"/> **"Extended"** (available to Enterprise): The previous month's "Compatible" release.

Reviewer: ```suggestion
- <Lifecycle status="coming soon"/> **"Extended"** (available to Enterprise): Provides a delayed release of the previous month's "Compatible" release.
```
